src/main/java/in/projecteka/datanotificationsubscription/subscription/model/SubscriptionResponse.java

```diff
@@ -0,0 +1,23 @@
+package in.projecteka.datanotificationsubscription.subscription.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class SubscriptionResponse {
```

Comment:
Can we keep the names similar to `ConsentFlow` as much as possible.
In consent flow we have:
- ConsentRequestRepresentation (The individual Request sent to frontend)
- ConsentRequestsRepresentation (The complete response object including filters etc)

This will help us sticking with a naming convention and would be easier to refer/find/search/debug.